### DISCUSSION TITLE
In Tab component, can position and orientation props work together?

### DISCUSSION THREAD

**Discussion by percy507** [mantinedev/mantine#1078] [Q&A] [answered]
After testing, it seems they can't work together. Looking forward to an official response.

> https://codesandbox.io/s/busy-bush-m8q7ro?file=/src/App.tsx

**Comment by rtivital** [comment, marked as answer]
Tabs with vertical orientation do not support position, you will need to customize styles with Styles API if you want this behavior

**Reply by percy507** [reply]
OK. I will do that with custom styles.
